README.md

```diff
@@ -1129,6 +1129,8 @@ first([
 //! "nope"
 ```
 
+When one Future rejects, the other gets cancelled automatically.
```

Comment:
This one should say *settles*. The `both` one now correctly states *rejects*. When cancellation happens is not the same between the two.

Reply:
Fixed